packages/react/src/lib/positioner/Popper.tsx

```diff
@@ -92,39 +123,50 @@ const Popper: React.FunctionComponent<PopperProps> = props => {
 
       const options: PopperJS.PopperOptions = {
         placement: proposedPlacement,
-        eventsEnabled,
         positionFixed,
         modifiers,
         onCreate: handleUpdate,
         onUpdate: handleUpdate,
       }
 
       popperRef.current = createPopper(targetRef.current, contentRef.current, options)
-      return () => popperRef.current.destroy()
     },
-    [computedModifiers, eventsEnabled, userModifiers, positionFixed, proposedPlacement],
+    [targetRef.current, contentRef.current, ...instanceDependencies],
```

Comment:
`targetRef.current`, `contentRef.current` are not dependencies of this function because it relies of `*Ref`s

Reply:
addressed, thanks